js/slideshow.js

```diff
@@ -103,6 +103,38 @@
 
 				// check if we moved along while we were loading
 				if (currentImageId === index) {
+                                        // check if not in cache
+                                        if (this.images[index].desc===undefined){
+                                            var params={location:this.images[index].path}
+                                            var url=OC.generateUrl('apps/gallery/files/exif?location={location}', params);
```

Comment:
Ah, yes, you're correct. I keep having to duplicate methods because we don't load too many files on the files side.